dev/src/bundle.ts

```diff
@@ -42,7 +40,7 @@ export class BundleBuilder {
   // The latest read time among all bundled documents and queries.
   private latestReadTime = new Timestamp(0, 0);
 
-  constructor(private bundleId: string) {}
+  constructor(public readonly bundleId: string) {}
```

Comment:
The TS style guide recommends using only "readonly" instead of "public readonly" as they are equivalent.